### ISSUE TITLE
Feature request: access console for successful requests

### ISSUE THREAD

**Issue by tigris** [BetterErrors/better_errors#151]
I find the ability to use the console on the error screen a god send. I seriously dunno how I lived without it until now.

Occassionally I find myself on the end of a successful request, in that it returns a 200 and renders the layout properly, but there may be things that didn't quite pan out the way I planned. Perhaps I got some logic in my view wrong etc.

I find a fantastic way to debug such things are to jump into the view and on the last line create some form of syntax error. Generally I will just <%= raise ArgumentError.new('debugging') %>. This will force better_errors to kick in at the last possible moment and I can step back through my app and have a console at any point I want to check certain things.

Whilst I realise I'm essentially using better_errors as a full blown IDE, it means I don't have to a) install some platform specific IDE like rubymine and b) be developing on the same machine that the code is running (in the case of developing remotely or via a VM).

Anyway, long story short, it would be great to be able to simply pass in some variable or hit up some url to emulate what I am doing here. In fact, I think I just convinced myself by writing this that one solution may be to have that ERB style raise error I gave above, at the bottom of my application layout around an if block for development? and params[:something].

**Comment by jekuno** [comment]
See https://github.com/charliesome/better_errors/issues/242#issuecomment-138521857
